simapp/app.go

```diff
@@ -108,27 +110,29 @@ type SimApp struct {
 	invCheckPeriod uint
 
 	// keys to access the substores
-	keys  map[string]*sdk.KVStoreKey
-	tkeys map[string]*sdk.TransientStoreKey
+	keys    map[string]*sdk.KVStoreKey
+	tkeys   map[string]*sdk.TransientStoreKey
```

Comment:
Whats the difference between "memStoreKey" and "transientStoreKey"?

Reply:
The memory store is in-memory only but persists across blocks, whereas the transient store is reset after each `Commit`, I believe.

Reply:
What @cwgoes stated is correct.